API/delivery/openapi.yaml

```diff
@@ -0,0 +1,283 @@
+          $ref: '#/components/schemas/Title'
+        required:
+          type: boolean
+          description: Whether a client displaying this element should force the user to fill it in before proceding
+      required:
+        - id
+        - type
+        - identifier
+    TextElement:
+      allOf:
+        - $ref: '#/components/schemas/Element'
+        - type: object
+          properties:
+            maxCharacterCount:
+              type: integer
+              minimum: -1
+              default: -1
+              maximum: 10000
+    RateableElement:
+      allOf:
+        - $ref: '#/components/schemas/Element'
+        - type: object
+          description: An element belonging to a form, that can contain multiple answer options
+          properties:
+            answerOptions:
+              type: array
+              items:
+                type: object
+                description: the answer options that rateable elements contains
+                properties:
+                  id:
+                    description: the UUID for the answer option
+                    allOf:
+                      - $ref: '#/components/schemas/Uuid'
+                  identifier:
+                    type: string
+                required:
+                  - id
+                  - identifier
+          required:
+            - answerOptions
+    ChoiceElement:
+      allOf:
+        - $ref: '#/components/schemas/RateableElement'
+        - type: object
+          description: An element belonging to a form, that can contain multiple answer options, and can allow for multiple answers to be selected by the end user
+          properties:
+            allowMultiple:
+              description: Indication whether this element allows multiple answer options to be selected
+              type: boolean
+          required:
+            - allowMultiple
+    NPSElement:
+      allOf:
+        - $ref: '#/components/schemas/RateableElement'
+        - type: object
+          description: An element that contains 11 answer options statically generated.
+          properties:
+            labelHigh:
+              $ref: '#/components/schemas/Label'
+            labelLow:
+              $ref: '#/components/schemas/Label'
+    SliderElement:
+      allOf:
+        - $ref: '#/components/schemas/Element'
+        - type: object
+          description: An element representing a set of values within a certain range
+          properties:
+            minimum:
+              type: integer
+              minimum: 0
+              maximum: 0
+            maximum:
+              type: integer
+              minimum: 1
+              maximum: 1000000
+            start:
+              type: integer
+              minimum: 0
+            labelHigh:
+              $ref: '#/components/schemas/Label'
+            labelLow:
+              $ref: '#/components/schemas/Label'
+            labelMiddle:
+              $ref: '#/components/schemas/Label'
+    FormConfiguration:
```

Comment:
The use of `Channel` vs `Form` is a little inconsistent. In the endpoint we pass a channel id to the channel resource, yet we return a form.

Reply:
Indeed it is a complex problem, since we query for a channel, which is a part of a form, to retrive the union of a channel and a form.

Reply:
I agree it might be a bit misleading. Usually, in the developer guide we refer to that schema as "form configuration" or "JSON configuration" (and their combinations).

Reply:
Yeah, we should probably agree on a name and make that consistent across the product and documentation.
I'd be in favour of renaming everything to `channel configuration`, but let's discuss this together.

Reply:
Note: we agreed on "channel configuration".